tox.ini

```diff
@@ -0,0 +1,41 @@
+[tox]
```

Comment:
Curious - Why the move out of pyproject.toml?

Reply:
I agree, I'd rather it stay in project.toml, despite It not being formatted so nicely there, part of the reason in so fond of the PEP for it is that it reduces the amount of tooling specific files in repo roots

Reply:
Makes sense 👍🏻 
Moved back to pyproject.toml